backend/src/main/java/com/woowacourse/levellog/team/domain/Participants.java

```diff
@@ -1,58 +1,66 @@
 package com.woowacourse.levellog.team.domain;
 
-import com.woowacourse.levellog.common.domain.BaseEntity;
 import com.woowacourse.levellog.common.support.DebugMessage;
-import com.woowacourse.levellog.member.domain.Member;
-import com.woowacourse.levellog.team.exception.ParticipantNotFoundException;
-import com.woowacourse.levellog.team.exception.ParticipantNotSameTeamException;
+import com.woowacourse.levellog.team.exception.HostUnauthorizedException;
+import com.woowacourse.levellog.team.exception.NotParticipantException;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import lombok.AccessLevel;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 
+@Embeddable
 @AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
 @Getter
 public class Participants {
 
-    private final List<Participant> values;
+    @OneToMany(mappedBy = "team", cascade = CascadeType.PERSIST, orphanRemoval = true)
+    private List<Participant> values = new ArrayList<>();
 
-    public List<Long> toInterviewerIds(final Long memberId, final int interviewerNumber) {
-        if (!isContains(memberId) || isWatcher(memberId)) {
-            return Collections.emptyList();
-        }
+    public static Participants of(final Team team, final ParticipantsIngredient ingredient) {
+        ingredient.validate(team.getInterviewerNumber());
+        final List<Participant> participants = new ArrayList<>();
 
```

Comment:
```suggestion
    public static Participants of(final Team team, final ParticipantsIngredient ingredient) {
    ingredient.toParticipants(team);
    }
```
`ParticipantsIngredient`에게 생성의 책임을 위임하는건 어떨까요?

Reply:
ParticipantsFactory에게 위임했습니다!